frontend/model/notifications/fakeDB.js

```diff
@@ -9,17 +9,17 @@ export default {
   },
   '1032': {
     timestamp: 1590822227327,
-    type: 'SEND_CONTRIBUTION',
+    type: 'CONTRIBUTION_REMINDER',
```

Comment:
Should this entire file be removed now?

Reply:
I don't know yet;  it might become useful as a basis for test data.

Reply:
Then those should be added in tests, and this file deleted.